service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantController.java

```diff
@@ -60,4 +62,12 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
 
         return respond(caseDetails);
     }
+
+    @PostMapping("/submitted")
+    public void handleSubmitted(@RequestBody CallbackRequest callbackRequest) {
+        publishEvent(new CaseDataChanged(getCaseData(callbackRequest)));
```

Comment:
You probs don't want to do this part if the state was not open or returned, task list is only visible in open and returned state I think